MatrixInterpolator.h

```diff
@@ -0,0 +1,143 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2021, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Description: Computes the MatrixInterpolator algorithm on the given snapshot matrix.
+
+#ifndef included_MatrixInterpolator_h
+#define included_MatrixInterpolator_h
+
+#include "Interpolator.h"
+#include <vector>
+#include <string>
+
+namespace CAROM {
+
+class Matrix;
+class Vector;
+
+/**
+ * MatrixInterpolator interpolates reduced matrices of a set of parameter points
+ * and returns an interpolated reduced matrix for an unseen parameter point.
```

Comment:
It is good that this general comment mentions that `MatrixInterpolator` is intended for reduced matrices only, but I would also recommend stating that performance depends on the matrix sizes being small, as the computational complexity is high, due to direct matrix inversion.